scripts/install_dev

```diff
@@ -12,9 +12,9 @@ if [ ! -d ~/catkin_ws/src/openag_brain/ ] ; then
   echo "Location \"~/catkin_ws/src/openag_brain\" does not exist. openag_brain must be placed in \"~/catkin_ws/src\" for this script to work." && exit 1
 fi
 ~/catkin_ws/src/openag_brain/scripts/init_dev || exit 1
+~/catkin_ws/src/openag_brain/scripts/install_db || exit 1
```

Comment:
My opinion is that couchdb related things shouldn't be part of `install_dev`, because the docker build is using docker compose to [provision the db in a separate container](https://github.com/OpenAgInitiative/openag_brain/pull/338/files#diff-4e5e90c6228fd48698d074241c2ba760R30), decoupling its management, and making it available in the brain container as `http://db:5984` (and to the outside world as well as `http://your-ip:5984`) 

If a single script installation is desired for non-docker builds, perhaps a new script that contained just the brain installation steps (something like `scripts/install_brain`) could be executed by both docker and the `install_dev` script (which would also run `install_db` to install couchdb)

Reply:
Ah, good point, thanks David.  I don't ever use docker (which is probably obvious).    I will make a new "developer_setup" script and take that line out of install_dev.